network/endpoint.go

```diff
@@ -52,6 +52,7 @@ type endpoint struct {
 	PODNameSpace             string `json:",omitempty"`
 	InfraVnetAddressSpace    string `json:",omitempty"`
 	NetNs                    string `json:",omitempty"`
+	SecondaryInterfaces      map[string]*InterfaceInfo
```

Comment:
can you add a comment indicating what's the key for this map.? I guess interface Name

Reply:
updated